interfaces.go

```diff
@@ -9,6 +9,11 @@ import (
 
 type MessageType = websocket.MessageType
 
+type WeightedBackend struct {
+	Handler RequestHandler
+	Weight  uint
+}
+
```

Comment:
Here we store types that need to be used across multiple packages to avoid/reduce  dependencies between modules. This type will be used only in context of `backend` package, by public type I meant just to name type with capital letter.

Reply:
ah I see will revert my last commit and fix it